uart/ping.toit

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2021 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the EXAMPLES_LICENSE file.
+
+import uart
```

Comment:
IMO this is slightly more complicated then needed. You can have a single UART echo to itself, by using pin 21 as TX and 22 as RX, and wire those two together. That way you don't need multiple UARTs running.

Reply:
I want to show a case where we use a full UART in an app.
If we only used the RX (resp. TX), then that would only show half of it.

Reply:
You'd still do this:
```
main:
  port := uart.....
  
  task::
    while line := reader.read_line:
      print line
      
  10.repeat:
    port.write "lala $it"
    sleep --ms=1000
```

I still believe this example is much easier to understand.

Reply:
Ok.
The main advantage of the current example is, that I can also show the `data := port.read`.
Will think how we can make things better. (Don't hesitate to ping me so I don't forget).